src/state/MediaViewModel.ts

```diff
@@ -387,8 +391,8 @@ abstract class BaseUserMediaViewModel extends BaseMediaViewModel {
     encryptionSystem: EncryptionSystem,
     livekitRoom: LivekitRoom,
     displayname$: Observable<string>,
```

Comment:
Why is this not becoming a Behavior as well?

Reply:
It did become a Behavior, maybe you're looking at the wrong commit?